docs/materials/motion/MotionPreview.css

```diff
@@ -0,0 +1,17 @@
+.motion-preview {
+    padding: 1rem 0;
+    margin-top: 2rem;
+}
+
+.object {
```

Comment:
Try to always scope your classes with a module even if it's only for docs, i've been using `o-ui-sb-`.